buildbot_ros_cfg/ros_deb.py

```diff
@@ -108,7 +108,7 @@ def ros_debbuild(c, job_name, packages, url, distro, arch, rosdistro, version, m
             ShellCommand(
                 haltOnFailure = True,
                 name = package+'-stampdeb',
-                command = ['git-dch', '-a', '--ignore-branch', '--verbose',
+                command = ['gbp', 'dch', '-a', '--ignore-branch', '--verbose',
```

Comment:
does this work with older distros as well. or do we need to fork for xenial? I don't have access to a test machine at the moment to check.

Reply:
gbp is also supported for trusty http://manpages.ubuntu.com/manpages/xenial/man1/gbp-buildpackage.1.html , I tried the command line and it is working for 14.04